client/js/widgets/dice.js

```diff
@@ -139,11 +139,19 @@ class Dice extends Widget {
     css += '; --pipColor:' + this.get('pipColor');
     css += '; --backgroundColor:' + this.get('backgroundColor');
 
-    const xRotations = [ 0, 90,  0,   0, -90,   0 ];
-    const yRotations = [ 0,  0, 90, -90,   0, 180 ];
-
-    css += '; --rotX:' + (xRotations[this.activeFace() % 6] + this.get('rollCount')*360) + 'deg';
-    css += '; --rotY:' + (yRotations[this.activeFace() % 6] + this.get('rollCount')*360) + 'deg';
+    if(this.get('classes') == 'd6') {
```

Comment:
There will be problems here if additional classes are used. That is one of the reasons I suggested `sides3d` property.

Reply:
Yeah, that's what I meant with my comment. The class names need to be there somewhere so we can target the CSS but it shouldn't be the user-facing property.